vm/string_test.go

```diff
@@ -29,6 +29,7 @@ func TestStringConversion(t *testing.T) {
 		{`"string".to_i`, 0},
 		{`"123string123".to_i`, 123},
 		{`"string123".to_i`, 0},
+		{`"Goby".to_a`, initArrayObject([]Object{initStringObject("G"), initStringObject("o"), initStringObject("b"), initStringObject("y")})},
```

Comment:
@st0012 Other string test cases works fine. Except, in the `String#to_a` test, when I tried to change the first element of expected value in `initArrayObject`, which is `initStringObject("G")` to other kind of value (e.g. `initStringObject("H")`), it didn't raise any error but passed the test